returnn/torch/engine.py

```diff
@@ -416,30 +426,29 @@ def run_forward_step(self, data: dict[str, torch.Tensor], run_ctx: RunCtx):
         with autocast(device_type=self._device, dtype=self._amp_dtype) if self._amp_dtype else nullcontext():
             self._forward_step_func(model=self._model, data=data, run_ctx=run_ctx, **sentinel_kw)
 
-    def _load_model(self, *, epoch: int, filename: Optional[str] = None):
+    def _load_model(self, *, epoch: int, filename: str):
```

Comment:
The type should stay `Optional`, because the filename is still optional as it can be `None` from the self.get_epoch_model(). (At least I hope so, otherwise we would even try to load models for a completely new training)